client/ayon_core/pipeline/publish/lib.py

```diff
@@ -252,17 +289,13 @@ def publish_plugins_discover(paths=None):
                 continue
 
             try:
-                module = import_filepath(abspath, mod_name)
+                module = _import_module(mod_name, Path(abspath))
```

Comment:
Why not update the implementation of `import_filepath` instead? I assume a similar fix may be well suited for other cases where we're importing things this way too?

E.g. is the same issue also present in our other plug-in systems currently, like our `Creators`, etc? because those also use `import_filepath` in a similar manner through the `modules_from_path` function. Or?

Reply:
done. I've also made it more safe for backwards compatibility.